src/modules/commander/Commander.cpp

```diff
@@ -1682,15 +1694,8 @@ void Commander::updateParameters()
 
 	int32_t value_int32 = 0;
 
-	// MAV_SYS_ID => vehicle_status.system_id
-	if ((_param_mav_sys_id != PARAM_INVALID) && (param_get(_param_mav_sys_id, &value_int32) == PX4_OK)) {
-		_vehicle_status.system_id = value_int32;
-	}
-
-	// MAV_COMP_ID => vehicle_status.component_id
-	if ((_param_mav_comp_id != PARAM_INVALID) && (param_get(_param_mav_comp_id, &value_int32) == PX4_OK)) {
-		_vehicle_status.component_id = value_int32;
-	}
+	// NOTE: The system can't handle changing the system_id, and component_id in the same session
```

Comment:
Comment needs to go with MAV_SYS_ID/MAV_COMP_ID.

It's also not that the system can't handle it, but that immediately on change (from a GCS) the GCS would need to be aware of the new SYS ID.